packages/dm-core/src/utils/arrayIntersection.tsx

```diff
@@ -0,0 +1,8 @@
+export const arrayIntersection = (
+  array1: string[],
+  array2: string[]
+): string[] => {
+  const result = array1.filter((item) => array2.includes(item))
+  console.log(array1, array2, result)
+  return result
+}
```

Comment:
I notice that we already have dependencies that use `lodash`, so we might as well use it ourselves. If we do, we don't have to write a lot of helper functions, like this one, and can instead just import them from lodash.

Reply:
Lets try lodash!